### PR TITLE
Updated Header Descriptions + Readded "do something cool" feature

### PR DESCRIPTION
**Pull request by joeyshi12**
plz merge

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+var compliments = ['You look great today', 'What\'s up cool guy?', 'Petar'];
```
Comment (102plee) [inline js/scripts.js:1]: While this line of code is alright, I think it's better to keep the "Joey is a fake gamer."

**Review by 102plee — CHANGES_REQUESTED** [review]
Overall a very nice addition. Please fix that one issue and it'll be good to go.

File: js/scripts.js (hunk 1 above)
Comment (joeyshi12) [reply js/scripts.js:1]: No

Comment (102plee) [reply js/scripts.js:1]: do it you coward

Comment (102plee) [reply js/scripts.js:1]: fine but get rid of petar

**Review by 102plee — APPROVED** [review]

**Review by 102plee — APPROVED** [review]

**Comment by joeyshi12** [comment]
yay

**Pull request merged**